src/components/Cards/Cards.module.scss

```diff
@@ -0,0 +1,148 @@
+@import '../../styles/variables.scss';
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+
+  &__item {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.card {
+  display: flex;
+  border: 1px solid $cardBorder;
+  border-radius: 1em;
+  margin-bottom: 2em;
+  max-width: 1000px;
+
+  &__img {
+    width: 268px;
+    height: 280px;
+    display: flex;
+    align-items: center;
+
+    & img {
+      width: 100%;
+      max-height: 280px;
+      object-fit: fill;
+    }
+  }
+
+  &__body {
+    margin: 32px 0 32px 32px;
+    width: 300px;
+  }
+
+  &__title {
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 27px;
+    color: $cardTitle;
+    margin: 0;
+  }
+
+  &__simpleText {
+    font-family: 'Open Sans';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 16px;
+    color: $cardSimpleText;
+    margin: 0;
+  }
+
+  &__stars {
+    color: $cardTitle  !important;
```

Comment:
Is this !important necessary here?

Reply:
I try not to use !important anywhere, but unfortunately, here I can't do without it , usually ant-design gives me the opportunity to change, but this time it can't